src/main/resources/liquibase/crop_changelog/16_1_0.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="medramos" id="v16.1.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="germplsm" columnName="germplsm_uuid"/>
+			</not>
+		</preConditions>
+		<comment>
+			Add guid column to germplsm table
+		</comment>
+		<addColumn tableName="germplsm">
+			<column name="germplsm_uuid" type="VARCHAR(36)">
+				<constraints nullable="true"/>
+			</column>
+		</addColumn>
+	</changeSet>
+
+	<changeSet author="mderamos" id="v16.1.0-2">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT
```

Comment:
Why does it need to drop this procedure when it's not yet created?

Reply:
It's for safety purposes. I'm not sure if it's the standard but I've seen multiple liquibase scripts doing the same, dropping a procedure before creating it.